intake_stac/catalog.py

```diff
@@ -331,15 +331,25 @@ def __init__(self, key, item, stacked=False):
     def _get_driver(self, entry):
         drivers = {
             'application/netcdf': 'netcdf',
+            'application/x-netcdf': 'netcdf',
+            'application/parquet': 'parquet',
+            'application/x-parquet': 'parquet',
+            # 'application/x-hdf5': '',
+            # 'application/x-hdf': '',
             'image/vnd.stac.geotiff': 'rasterio',
             'image/vnd.stac.geotiff; cloud-optimized=true': 'rasterio',
             'image/x.geotiff': 'rasterio',
+            'image/tiff; application=geotiff': 'rasterio',
+            'image/tiff; application=geotiff; profile=cloud-optimized': 'rasterio',  # noqa: E501
             'image/png': 'xarray_image',
             'image/jpg': 'xarray_image',
             'image/jpeg': 'xarray_image',
             'text/xml': 'textfiles',
             'text/plain': 'textfiles',
             'text/html': 'textfiles',
+            'application/geo+json': 'geopandas',
+            # 'application/json': '?',
+            # 'application/geopackage+sqlite3': 'geopandas',
```

Comment:
I don't think intake has a driver for this yet. I suppose it would be geopandas once available though.

Reply:
Should we leave it commented out, or just not include for now?

Reply:
Right so the question is do we leave the stubs or just note it somewhere else like in tickets?

Reply:
i'm in favor of leaving the comments in and making a not in an open issue